map/src/test/java/openlr/map/simplemockdb/OpenLRMapDatabaseAdaptor.java

```diff
@@ -0,0 +1,137 @@
+package openlr.map.simplemockdb;
+
+import com.vividsolutions.jts.geom.Envelope;
+import com.vividsolutions.jts.geom.Coordinate;
+import com.vividsolutions.jts.geom.LineString;
+import generated.SimpleMockedMapDatabase;
+import openlr.map.GeoCoordinates;
+import openlr.map.Line;
+import openlr.map.MapDatabase;
+import openlr.map.Node;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Unmarshaller;
+import java.awt.geom.Rectangle2D;
+import java.io.InputStream;
+import java.util.*;
+
+public class OpenLRMapDatabaseAdaptor implements MapDatabase {
+    private Map<Long, SimpleMockedNode> nodes = new HashMap<>();
+    private Map<Long, SimpleMockedLine> lines = new HashMap<>();
+    private SimpleMockedMapDatabase simpleMockedMapDatabase;
```

Comment:
I think you are not doing a proper wrapper of the `SimpleMockedMapDatabase` because it is only used in the initialization phase. You should separate the construction logic with the implementation of the `MapDatabase`. Same for `SimpleMockedLine`and `SimpleMockedNode`

Reply:
cc0fcab1679ad3d5ae77d3fb09c44fc5646d400c

Reply:
78bdb5587551a7911bb13512302853adcb9c529c